src/net.cpp

```diff
@@ -1393,8 +1394,9 @@ void ThreadOpenConnections()
         if (GetTime() - nStart > 60) {
             static bool done = false;
             if (!done) {
-                // skip DNS seeds for staked chains.
-                if ( is_STAKED(ASSETCHAINS_SYMBOL) == 0 ) {
+                // skip DNS seeds for staked chains. 
+                // Also skip fixed kmd seeds for assets chains
+                if ( is_STAKED(ASSETCHAINS_SYMBOL) == 0 && !IS_ASSET_CHAIN(ASSETCHAINS_SYMBOL)) {
```

Comment:
This is an example of what I am talking about in [this comment](https://github.com/KomodoPlatform/komodo/pull/520#issuecomment-1016489222). `!IS_ASSET_CHAIN(ASSETCHAINS_SYMBOL)` is much clearer than `ASSTECHAINS_SYMBOL[0] != 0` in my opinion. :1st_place_medal: 

And if we could get to the point where calls like this were `constexpr`, they would be 0 runtime cost. I can dream, can't I?